components/VM_Arm/src/main.c

```diff
@@ -188,11 +188,11 @@ static int _dma_morecore(size_t min_size, int cached, struct dma_mem_descriptor
     dma_desc->vaddr = _vaddr;
     dma_desc->paddr = getaddr_ret.paddr;
     dma_desc->cached = 0;
-    dma_desc->size_bits = 12;
+    dma_desc->size_bits = seL4_PageBits;
     dma_desc->alloc_cookie = (void *)frame;
     dma_desc->cookie = NULL;
     /* Advance the virtual address marker */
-    _vaddr += BIT(12);
+    _vaddr += seL4_PageBits;
```

Comment:
`BIT(seL4_PageBits)`?

Reply:
Thanks for spotting this, seems like my tired eyes missed something. Will fix.

Reply:
Fixed now.